src/app/home/page.tsx

```diff
@@ -47,22 +48,32 @@ interface HomeLoggedInProps{
     logout: EventHandler<MouseEvent<HTMLButtonElement>>
 }
 function HomeLoggedIn(props: HomeLoggedInProps) {
-    const {username, userId, puzzleRating} = useSelector(selectActiveUserLichessData)
+    const {username, userId: lichessUserId, puzzleRating} = useSelector(selectActiveUserLichessData)
+    const eventSocketService = useEvents()
+    const userId = useSelector(selectActiveUserId)
     const [loginUser, { data, loading, error }] = useMutation(LOGIN_USER)
     useEffect(() => {
-        if(username && userId && puzzleRating) {
-            console.log("Here")
+        if(username && lichessUserId && puzzleRating) {
         loginUser({
             variables: {
                 userData: {
-                "lichessId": userId,
+                "lichessId": lichessUserId,
                 "lichessUsername": username,
                 "lichessPuzzleRating": puzzleRating
                 }
             }
         })
         }
-    }, [username, userId, puzzleRating]);
+    }, [username, lichessUserId, puzzleRating]);
+    useEffect(() => {
+        if(userId) {
+            eventSocketService.notifyLogin(userId)
+            return eventSocketService.on(
+                'GameStart',
```

Comment:
What's this?

Reply:
setting up the websocket, when the game starts, we should redirect to the game, this is mostly just testing and playing around for now